independent-publisher-connectors/Star Wars/apiProperties.json

```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#da3b01",
+    "capabilities": [],
+    "publisher": "Paul Culmsee",
+	"stackOwner": "https://swapi.dev/"

```

Comment:
@paulculmsee Please remove the extra space.

Reply:
@paulculmsee Sorry for the delay in response, Can you please resolve this comment to get your PR approved and merged.

Reply:
Sorry mate. Slowly getting better at this